build.sbt

```diff
@@ -24,12 +26,21 @@ lazy val core = project
 
     quasarPluginQuasarVersion := IO.read(file("./quasar-version")).trim,
 
-    quasarPluginDestinationFqcn := Some("quasar.datasource.postgres.PostgresDestination$"),
+    quasarPluginDestinationFqcn := Some("quasar.plugin.postgres.PostgresDestinationModule$"),
 
     /** Specify managed dependencies here instead of with `libraryDependencies`.
       * Do not include quasar libs, they will be included based on the value of
       * `quasarPluginQuasarVersion`.
       */
     quasarPluginDependencies ++= Seq(
+      "org.slf4s"    %% "slf4s-api"       % "1.7.25",
+      "org.tpolecat" %% "doobie-core"     % DoobieVersion,
+      "org.tpolecat" %% "doobie-hikari"   % DoobieVersion,
+      "org.tpolecat" %% "doobie-postgres" % DoobieVersion,
+
+      "com.github.tototoshi" %% "scala-csv" % "1.3.6" % Test,
+      "com.slamdata" %% "qdata-core" % "10.0.31" % Test,
+      "com.slamdata" %% "quasar-foundation" % quasarPluginQuasarVersion.value % "test->test" classifier "tests",
+      "io.argonaut" %% "argonaut-scalaz" % "6.2.3" % Test
```

Comment:
Are we supposed to put test dependencies in here?

Reply:
Yeah, it is just appended to `libraryDependencies`. I guess we don't actually _need_ the `quasarPluginDependencies` key, we could just leave the comment about not including quasar deps.

Reply:
Doesn't it get included in the assembly as well though?

Reply:
Ah, right they do, that’s what the additional key is for, will fix.